src/crud/workspace.py

```
@@ -0,0 +1,97 @@
+
+    Raises:
+        ConflictException: If there's an integrity error when creating the workspace
+    """
+    # Try to get the existing workspace
+    stmt = select(models.Workspace).where(models.Workspace.name == workspace.name)
+    result = await db.execute(stmt)
+    existing_workspace = result.scalar_one_or_none()
+
+    if existing_workspace is not None:
+        # Workspace already exists
+        logger.debug(f"Found existing workspace: {workspace.name}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)